### DISCUSSION TITLE
Sidee Howsha lo qeybsanaa

### DISCUSSION THREAD

**Discussion by ducesane** [dugsiiyeinc/dhaqsoPay#11] [General]
Asc

Asxaabta Extention-ka aan isla sameeheeno sida ayaa u qeybsananaa  dhisidiiisa , 
@sharafdin  xage ka helenaa google chrome extensions start kit

**Comment by sharafdin** [comment]
Wcs, akhii marka hore design-ka aan isku raacno, kadibna adinkaa ku heshiinaayo oo project board ka ayaad ku maamulaysaan, dhanka starter-kana aniga ayaa soo habeenaayo Insha'Allah!

**Reply by ducesane** [reply]
Waa war fiican

**Reply by sharafdin** [reply]
eng hadeer waa diyaar starter kit lakin kahor aan ka hormarsano xiisad yar Insha'Allah!

**Reply by ducesane** [reply]
inash allah waa sugeynaa

**Comment by hayle102** [comment]
Asxaabey hadda wax walba waa diyaar haddi ay ahaan lahayd design, iyo sida loo qaybsaday shaqada waxaadna ka heleysaa
design:
#42 

sidoo kale waxaa lagu qaybiyay shaqada issue waxaad tagtaa project kadib meesha ayuu ugu diyaarsanyahayq qof waliba iyo shaqada uu qaban lahaa iyada oo faahfaahsan.
